src/Tools/ExternalAccess/FSharp/Internal/FSharpContentTypeDefinitions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.ExternalAccess.FSharp.Editor;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.FSharp.Internal
+{
+    internal static class FSharpContentTypeDefinitions
+    {
+        [Export]
+        [Name(FSharpContentTypeNames.FSharpContentType)]
+        [BaseDefinition(FSharpContentTypeNames.RoslynContentType)]
+        public static ContentTypeDefinition FSharpContentTypeDefinition => new ContentTypeDefinition();
```

Comment:
```suggestion
        public static ContentTypeDefinition FSharpContentTypeDefinition;
```

Reply:
Why make this change?

Reply:
Ah ok, looks like I can do this considering every other example using `[BaseDefinition]` doesn't have that accessor.

Reply:
Will make this `readonly` as well.